src/UserForm.test.js

```diff
@@ -0,0 +1,224 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import UserForm from './UserForm';
+
+jest.mock('@folio/stripes/components', () => ({
+  Col: ({ xs, children }) => (
+    <div data-testid="col" xs={xs}>{children}</div>
+  ),
+  KeyValue: ({ label, value }) => (
+    <div>
+      <span>{label}:</span>
+      <span data-testid="key-value">{value}</span>
+    </div>
+  ),
+  Row: ({ children }) => (
+    <div data-testid="row">{children}</div>
+  ),
+  Select: ({ name, label, onChange, required, children, value }) => {
+    return (
+      <div>
+        <label htmlFor={name}>{label}</label>
+        <select
+          id={name}
+          name={name}
+          onChange={onChange}
+          required={required}
+          data-testid="select"
+          value={value}
+        >
+          {children}
+        </select>
+      </div>
+    );
+  },
+}));
+
+jest.mock('react-final-form', () => ({
+  Field: ({ name, label, component, onChange, required, validate, children }) => {
+    const Component = component;
+    return (
+      <div>
+        <label htmlFor={name}>{label}</label>
+        <Component
+          id={name}
+          name={name}
+          onChange={onChange}
+          required={required}
+          validate={validate}
+        >
+          {children}
+        </Component>
+      </div>
+    );
+  },
+}));
+
+jest.mock('@folio/stripes/smart-components', () => ({
+  ProxyManager: () => (
+    <div data-testid="proxy-manager" />
+  ),
+}));
```

Comment:
We have separate files with mocks here - test/jest/__mock__.
You can use them and if you do not find an appropriate mock you can add it there.